### DISCUSSION TITLE
CSS style issue

### DISCUSSION THREAD

**Discussion by Yedidya10** [community/community#38589] [Other Feature Feedback, Questions, & Ideas]
Hello.

As you can see in the picture, there is a dividing line that should not appear when you open the Recent Repositories list.
It seems that it is recommended to apply `overflow: scroll` to this column.
![image w](https://user-images.githubusercontent.com/86730512/200629077-d19e8724-c5f8-4bdf-9801-f7efc70c1683.png)

<br/>
<br/>

Also, in the Chrome browser, at the end of the Recent Repositories column, this section does not appear or appears incorrectly:
![image](https://user-images.githubusercontent.com/86730512/200629223-3b9efe22-9dd0-42c0-9a37-b11c13e21e9d.png)

**Comment by ossanna16** [comment]
Hi there @Yedidya10 and welcome to our community! Thank you for the feedback!
